FogCreek/Glitch-Community
# Necessary part: componentize questions

## Links
* Remix link: https://necessary-part.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328434/

## GIF/Screenshots:
![Screen Shot 2019-04-26 at 6 08 26 PM](https://user-images.githubusercontent.com/938722/56838975-5ee03300-684e-11e9-8b9f-4d44d8f066c5.png)

## Changes:
* create Questions component
* delete old Questions presenter & styles

## How To Test:
* visit https://necessary-part.glitch.me/
* verify the "nobody is asking for help" kaomoji updates every time the timer completes
* create a new project and ask for help (TODO: would be nice if this weren't visible to _every_ user, but  ¯\_(ツ)_/¯)
* verify that your question appears on the home page and on https://necessary-part.glitch.me/questions
* for more detailed design analysis, view the storybook at https://necessary-part.glitch.me/storybook/index.html?path=/story/questions--has-questions
